docs/api/routers/Routers.md

```diff
@@ -78,7 +78,7 @@ const MyAppRouter = {
   getStateForAction(action, state) {
     if (
       state &&
-      action.type === 'Back' &&
+      action.type === actions.BACK &&
```

Comment:
This is purely visual. But I think we should format the names as `NavigationActions.Back` etc. All lowercase prefix and uppercase action name looks very weird.